src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -284,9 +286,9 @@ Start, Stop, Restart
       This sets the :term:`final cycle point` six hours after the
       :term:`initial cycle point`.
 
-      Run `cylc validate` to check for any errors::
+      Run `cylc validate` to check for any errors:
```

Comment:
The double colon is actually an RST feature, it is short-hand for `.. code-block::`:

```suggestion
      Run `cylc validate` to check for any errors::
```

Reply:
Oh right. For some reason, the [rst extension for VSCode](https://github.com/vscode-restructuredtext/vscode-restructuredtext) doesn't like that; the syntax highlighting for everything after that line doesn't work